include/aikido/rviz/InteractiveMarkerViewer.hpp

```diff
@@ -34,22 +41,29 @@ class InteractiveMarkerViewer
 
   interactive_markers::InteractiveMarkerServer& marker_server();
 
-  SkeletonMarkerPtr addSkeleton(const dart::dynamics::SkeletonPtr& skeleton);
-
-  FrameMarkerPtr addFrame(
+  /// Visualizes a Skeleton.
+  /// \param skeleton Skeleton to add to the viewer
+  /// \return Skeleton marker added to the viewer.
+  SkeletonMarkerPtr addSkeletonMarker(
+      const dart::dynamics::SkeletonPtr& skeleton);
+
+  /// Visualizes a Frame as a cylinder.
+  /// \param[in] frame Target DART frame.
+  /// \param[in] length Length of the cylinderical frame (along z-axis).
+  /// \param[in] thickness Radius (thickness in x and y axes) of cylinder.
+  /// \param[in] alpha Opacity.
+  /// \return Frame marker added to this viewer.
+  FrameMarkerPtr addFrameMarker(
       dart::dynamics::Frame* frame,
       double length = 0.25,
       double thickness = 0.02,
       double alpha = 1.0);
 
-  SkeletonMarkerPtr CreateSkeletonMarker(
```

Comment:
It looks like a handy helper for creating `SkeletonMarker`. You can use it in `addSkeleton` and `update`. Why bother removing it?

Reply:
I preferred consistency (across skeletons, frames etc.) over code duplication. We could also have `createFrameMarker`, `createTSRMarker` etc. I just removed it instead.